kumascript/macros/GlossarySidebar.ejs

```diff
@@ -0,0 +1,32 @@
+<%
+var locale = env.locale;
+var baseURL = '/' + locale + '/docs/Glossary';
+
+var text = mdn.localStringMap({
+  'en-US': {
+    'Glossary_home_page' : 'MDN Web Docs Glossary',
+  },
+  'es': {
+    'Glossary_home_page' : 'Glosario de MDN Web Docs',
+  },
+  'fr': {
+    'Glossary_home_page' : "Glossaire MDN Web Docs",
+  },
+  'ja': {
+    'Glossary_home_page' : 'MDN Web Docs 用語集',
+  },
+  'ko': {
+    'Glossary_home_page' : 'MDN Web Docs 용어집',
+  },
+  'ru': {
+    'Glossary_home_page' : 'Глоссарий MDN Web Docs',
+  }
+});
+
+%>
+
+<section id="Quick_links" data-macro="GlossarySidebar">
+  <ol>
+    <li><strong><a href="<%=baseURL%>"><%=text['Glossary_home_page']%></a></strong><%-await template("ListSubpagesForSidebar", ['/en-us/docs/Glossary', 1])%></li>
```

Comment:
The parent `<li>` doesn't usually contain the subtree in other sidebars (and with the `localString` above you can reference it as `text` instead):

```suggestion
    <li><strong><a href="<%=baseURL%>"><%=text%></a></strong></li>
    <%-await template("ListSubpagesForSidebar", ['/en-us/docs/Glossary', 1])%>
```

Reply:
To avoid the `localString()` at all, use these helper functions:

```js
async function renderRootItem(slug) {
  const [link, title] = await getPageLinkAndTitle(slug);
  return `<li><a href="${link}"><strong>${title}</strong></a></li>`
}

async function getPageLinkAndTitle(slug) {
  let link = `/${env.locale}${slug}`;
  let page = await wiki.getPage(link);
  if (!page.title && env.locale !== 'en-US') {
    link = `/en-US${slug}`;
    page = await wiki.getPage(link);
  }
  let title = page.short_title || page.title;
  title = mdn.htmlEscape(title);
  return [link, title];
}
```

And then you can replace it as follows:

```suggestion
    <%- await renderRootItem("/en-us/docs/Glossary") %>
    <%- await template("ListSubpagesForSidebar", ['/en-us/docs/Glossary', 1]) %></li>
```

Reply:
I would suggest to set `short-title: "Glossary"` here:

https://github.com/mdn/content/blob/a7d6d3d100d2bf389b47ec70d59b30dbf2069403/files/en-us/glossary/index.md#L1-L5

Reply:
Done in https://github.com/mdn/yari/pull/8997/commits/880c89fb50d5cc88adfaa0dcd6ae7527827f910e

and in content at https://github.com/mdn/content/pull/26985/commits/428117974b13e9df915c9dc8ad22fd505eda84a3